addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/EnvisalinkBridgeHandler.java

```diff
@@ -120,7 +120,7 @@ public void openConnection() {
             logger.error("openConnection(): IO Exception: ", ioException);
             setConnected(false);
         } catch (Exception exception) {
-            logger.error("openConnection(): Exception: ", exception);
+            logger.error("openConnection(): Unable to open a connection: ", exception);
```

Comment:
Just to mention it (as this appears throughout your code): The much better way is to simply add the message of the exception (in cases where the stacktrace is not relevant):
```
            logger.error("openConnection(): Unable to open a connection: {}", exception.getMessage());
```
If the stacktrace is relevant, you can do a
```
            logger.error("openConnection(): Unable to open a connection: {}", exception.getMessage(), exception);
```